salt/states/pkg.py

```diff
@@ -3399,6 +3406,45 @@ def group_installed(name, skip=None, include=None, **kwargs):
             This option can no longer be passed as a comma-separated list, it
             must now be passed as a list (as shown in the above example).
 
+    .. note::
+        The below options are only supported on RPM-based systems
+
+    fromrepo
+        Restrict ``yum groupinfo`` to the specified repo(s).
+        (e.g., ``yum --disablerepo='*' --enablerepo='somerepo'``)
+
+        .. code-block:: yaml
+
+            MyGroup:
+              pkg.group_installed:
+                - fromrepo: base,updates
+
+        .. versionadded:: 3006.2
```

Comment:
So, if on https://github.com/saltstack/salt/pull/64349/files#r1210742029 we can reasonable say it's a bug, because of your argument, why is it `versionadded` here?

Reply:
Because support for these arguments needed to be added to make the code agree with the documentation: https://github.com/saltstack/salt/blob/c33b562/salt/states/pkg.py#L3402-L3406

Is it no longer considered a bug when the code doesn't do what the documentation says? I'm confused.

Reply:
> Is it no longer considered a bug when the code doesn't do what the documentation says? I'm confused.

It is, of course.

But can I argue that the function changed to accommodate the missing(and promised support) for the extra arguments?

Reply:
I was suggesting from `versionadded` to `versionchanged` not that this is not code added to fix a bug.

Ignore my rant, I agree with the `versionadded` usage here.